ui-admin/src/study/participants/enrolleeView/EnrolleeProfile.tsx

```diff
@@ -96,13 +96,22 @@ export function ReadOnlyProfile(
   }
 ) {
   const mailingAddress = profile.mailingAddress
+
   return <>
-    <ReadOnlyRow title={'Name'} values={[`${profile.givenName} ${profile.familyName}`]}/>
+    <ReadOnlyRow title={'Name'}
+      values={[
+        !isEmpty(profile.givenName) || !isEmpty(profile.familyName)
+          ? `${profile.givenName || ''} ${profile.familyName || ''}`
+          : ''
+      ]}/>
     <ReadOnlyRow title={'Birthdate'} values={[dateToDefaultString(profile.birthDate)]}/>
     <ReadOnlyRow title={'Primary Address'} values={[
       mailingAddress.street1,
       mailingAddress.street2,
-      `${mailingAddress.city}, ${mailingAddress.state} ${mailingAddress.postalCode}`
+      (!isEmpty(mailingAddress.city) || !isEmpty(mailingAddress.state) || !isEmpty(mailingAddress.postalCode))
```

Comment:
this is kind of gross -- and addresses are only going to get more complicated over time.  I'd just make a separate "ReadOnlyAddress" component

Reply:
I agree - I will fix that as part of this PR.